### DISCUSSION TITLE
Is there any support for Waveshare RS485 CAN HAT for Raspberry Pi

### DISCUSSION THREAD

**Discussion by azsoccerpop** [tagyoureit/nodejs-poolController#902] [RS485]
I am looking for control of my Intelliflo VS pump using Nixie as I do not have any Pentair automation equipment. I know there are recommendations for USB RS485 but I was hoping to use this Hat on my RPi4b inside of a very tight Intellimatic box once the mechanical clock is removed.

**Comment by rstrouse** [comment]
I don’t know if that card simply connects to the existing tty or not but the sequent 8 relay has an rs485 on it that will work.

**Reply by tagyoureit** [reply]
I've used a Waveshare hat before.  You have to install a driver for it, but then it shows up like a regular port you can specify in njsPC.

**Reply by azsoccerpop** [reply]
So I beleive I have found the answer which is an edit to enable UART via the /boot/config.txt with "enable_uart=1" I now see /dev/ttS0 as a usable port. Hopefully I can connect and test tomorrow with my Intelliflo VS when I get the cable.

**Comment by azsoccerpop** [comment]
I can confirm that the Waveshare RS485 CAN HAT for Raspberry Pi is recognized as Comms Port /dev/ttyS0 after editting the /boot/config.txt with "enable_uart=1". I am now able to control my Intelliflo VS pump via Nixie Single Body.
